Docs: organization docs refactor (#46812)
grafana/grafana: docs/sources/administration/manage-organizations/_index.md

---
title: 'Managing organizations'
menuTitle: 'Managing organizations'
description: 'Describes how to use organizations to isolate dashboard to users and teams.'
weight: 300
aliases: [docs/sources/administration/manage-organizations/_index.md]
keywords:
  - organizations
  - dashboards
---

# Managing organizations

This topic describes what organizations are, and how to create, edit, and delete them.

## About organizations

> An organization is an entity that helps you limit the dashboards that users can see. Their purpose is to provide completely separate experiences, which look like multiple instances of Grafana, within a single instance. Multiple organizations are easier and cheaper to manage than multiple instances of Grafana.

Reviewer: ```suggestion
An organization is an entity that helps you isolate users and resources like dashboards, annotations, and data sources from each other. Their purpose is to provide completely separate experiences, which look like multiple instances of Grafana, within a single instance. Multiple organizations are easier and cheaper to manage than multiple instances of Grafana.
```

> To an organization you can add teams and dashboards. The member of one organization cannot view dashboards assigned to another organization. However, a user can belong to multiple organizations.

Reviewer: ```suggestion
Users, configuration settings, and Grafana Enterprise licenses are shared between organizations. Other resources, like dashboards, data sources, annotations, folders, Teams, and Alerts, are isolated within each organization and cannot be easily shared with another organization. The member of one organization cannot view dashboards assigned to another organization. However, a user can belong to multiple organizations.
```

Grafana Server Administrators are responsible for creating organizations. For more information about the Grafana Server Administrator role, refer to [Grafana server administrators]({{< relref "../manage-users-and-permissions/about-users-and-permissions.md#Grafana server administrators" >}}).

## View a list of organizations

Complete this task when you want to view a list of existing organizations.

### Before you begin

- Ensure that you have Grafana Server Administrator permissions

**To view a list of organizations:**

1. Sign in to Grafana as a server administrator.
1. Hover your cursor over the **Server Admin** (shield) icon until a menu appears, and click **Orgs**.

A list of organizations appears.

![Server Admin organization list](/static/img/docs/manage-users/server-org-list-7-3.png)

## Create an organization

> Create an organization when you want to segment dashboards among Grafana users.

Reviewer: ```suggestion
Create an organization when you want to isolate dashboards and other resources from each other.
```
